### DISCUSSION TITLE
Make kedder's openvario-shell default

### DISCUSSION THREAD

**Discussion by lordfolken** [Openvario/meta-openvario#206] [Ideas]
Reference: https://awesomeopensource.com/project/kedder/openvario-shell

I think his menu has a better look, better functionality (for example WiFi configuration).
Also the extensibility is a good point.
For example the https://github.com/kedder/openvario-compman competition manager for soaringspot.com.

**Comment by August2111** [comment]
@lordfolken Please have a look at discussion to PR #197, it's about the future of Python and also Kedders shell...

**Comment by kedder** [comment]
@lordfolken I was planning to officially "propose" after I do a file transfer app, something several users requested because it wasn't obvious enough that you can transfer files with backup/restore tool. That effort has stalled a bit, but I'll try to find time for this.

**Reply by DanD222** [reply]
> @lordfolken I was planning to officially "propose" after I do a file transfer app, something several users requested because it wasn't obvious enough that you can transfer files with backup/restore tool. That effort has stalled a bit, but I'll try to find time for this.

I got the point, I just didn’t think it was a particularly elegant way of doing things ;) In any case, I’m grateful for ov-shell in the as-is state, and I don’t want to bend your arm to work on things you perhaps don’t see value in. Plus there’s always the option to use syncthing as an alternative route for copying individual files into the XCSoar home directory.
